.github/workflows/L2-ContentProtection.yml

```
@@ -0,0 +1,35 @@
+      - uses: actions/checkout@v4
+        with:
+          path: ${{github.repository}}
+
+      - name: Install cmake
+        run: |
+          sudo apt update
+          sudo apt install -y cmake
+
+      - name: Build Thunder
+        working-directory: ${{github.workspace}}
+        run: sh +x ${GITHUB_REPOSITORY}/.github/workflows/BuildThunder.sh
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:7: Double quote to prevent globbing and word splitting

(shellcheck)